spacy/pipeline/transition_parser.pyx

```diff
@@ -359,7 +361,42 @@ class Parser(TrainablePipe):
 
     def rehearse(self, examples, sgd=None, losses=None, **cfg):
         """Perform a "rehearsal" update, to prevent catastrophic forgetting."""
-        raise NotImplementedError
+        if losses is None:
+            losses = {}
+        for multitask in self._multitasks:
+            if hasattr(multitask, 'rehearse'):
+                multitask.rehearse(examples, losses=losses, sgd=sgd)
+        if self._rehearsal_model is None:
+            return None
+        losses.setdefault(self.name, 0.)
```

Comment:
```suggestion
        losses.setdefault(self.name, 0.0)
```
(I think we've been slightly inconsistent in the past but should probably standardize on `0.0`?)